### DISCUSSION TITLE
Can photos timeline be grouped in month rather than day?

### DISCUSSION THREAD

**Discussion by kchen0x** [immich-app/immich#1626] [Feature Request]
### Feature detail

![6B541097-AAE8-4E9B-9E08-CBC892904069](https://user-images.githubusercontent.com/9739009/209426691-5cc7a6f2-7535-4566-9330-696faaa4a187.png)

Too much margin on the right if I take only one photo in the day. It could be better to group photos in month, or zoom in and out to set group range from day to year.

### Platform

Mobile App

**Comment by jrasm91** [comment]
Implemented in #1563.
